spec/feature/change_phone_spec.rb

```diff
@@ -17,7 +17,7 @@
     enter_mfa
 
     expect(page).to have_text(I18n.t("account.manage.heading"))
-    expect(user.reload.phone).to eq(new_phone_number)
+    expect(user.reload.phone).to eq("+447581123456")
```

Comment:
How come this changed from a variable hard coded?
Are we no longer using this elsewhere - or are you thinking this is clearer to read?

Reply:
We're expecting the phone number to be transformed from the UK format to the international format.  Since the international format version of the number is only being used in one place, it didn't seem necessary to assign the value to a variable.